typings/pinejs-client-core.d.ts

```diff
@@ -70,7 +70,7 @@ interface Lambda<T> {
 }
 
 type OrderByValues = 'asc' | 'desc';
-type OrderBy = string | string[] | { [index: string]: OrderByValues };
+type OrderBy = string | OrderBy[] | { [index: string]: OrderByValues };
```

Comment:
Does it really allow recursive nested arrays?
I'm fine with this as is though for now, since it will error while developing.

Reply:
It allows it, basically if it sees an array it just loops through it and recurses. I wouldn't recommend it but it is allowed

Reply:
Nice :+1: